mlflow/java/spark/src/main/scala/org/mlflow/spark/autologging/MlflowAutologEventPublisher.scala

```diff
@@ -42,15 +43,23 @@ private[autologging] trait MlflowAutologEventPublisherImpl {
       "autologging."))
   }
 
+  // Exposed for testing
+  private[autologging] def getSparkSessionUUID: String = {
+    spark.getClass().getDeclaredMethod("sessionUUID").invoke(spark).toString
```

Comment:
I've verified that all REPL-aware environments we work with (i.e. Databricks REPLs) have this property available (i.e MLR 6.4+).

Reply:
why not use `spark. sessionUUID` but use invoke here ?